pkg/preparer/orchestrate.go

```diff
@@ -418,7 +420,9 @@ func (p *Preparer) installAndLaunchPod(pair ManifestPair, pod Pod, logger loggin
 			}
 		} else {
 			// TODO: do this in a transaction
-			err = p.podStore.WriteRealityIndex(pair.PodUniqueKey, p.node)
+			// TODO dai - what do I do with the errors?
+			ctx, cancelFunc := transaction.New(context.Background())
```

Comment:
also it's harmless if `cancelFunc()` is called twice, which it will be in the happy path.